README.md

```diff
@@ -1,65 +1,283 @@
+
+Run the following command to install the MinIO Operator and Plugin using `krew`:
+
+```sh
+   kubectl krew update
+   kubectl krew install minio
+```
+
+Run the following command to initialize the Operator:
 
 ```sh
 kubectl minio init
+
 ```
 
-Once the MinIO Operator is created, proceed with Tenant creation.
+## 2) Create a New Tenant
 
-## Tenant Setup
+The following `kubectl minio` command creates a MinIO Tenant with 4 nodes, 16
+volumes, and a total capacity  of 16Ti. This configuration requires
+*at least* 16 
+[Persistent Volumes](https://github.com/minio/operator#Local-Persistent-Volumes).
 
-A Tenant is a MinIO cluster created and managed by the Operator. Before creating tenant, please ensure you have requisite nodes and drives in place and relevant PVs are created.
+```sh
+   kubectl minio tenant create minio-tenant-1 \
+      --servers 4                             \
+      --volumes 16                            \
+      --capacity 16Ti                         \
+      --namespace minio-tenant-1              \
+      --storageClassName local-storage        \
+```
 
-In below example, we ask MinIO Operator to create a Tenant yaml with 4 nodes, 16 volumes, and 16 Ti total raw capacity (4 volumes of 1 Ti per node). This means you need to have 4 PVs of 1Ti each per node, with a total of 4 nodes, before attempting to create the MinIO tenant.
+- The `minio-tenant-1` argument specifies the name of the MinIO Tenant. The MinIO
+  Operator uses this name as a prefix for certain resources in the Tenant.
 
-Since MinIO has built-in erasure-code support for high-availability and data protection, we recommend the following CSI drivers for high-performance and scalability:
+- The `--servers` field indicates the number of `minio` pods to deploy into the cluster.
+  The cluster *must* have at least one available Node per `minio` pod. 
 
-- [Local PV](https://kubernetes.io/docs/concepts/storage/volumes/#local)
-- [OpenEBS Local PV](https://docs.openebs.io/docs/next/localpv.html)
-- [vSAN Direct](https://blogs.vmware.com/virtualblocks/2020/09/15/whats-new-in-vsan-7-update-1/)
+- The `--volumes` field indicates the total number of volumes in the Tenant. MinIO
+  generates a Persistent Volume Claim (`PVC`) for each volume and evenly distributes
+  volumes across each `minio` pod. The example above results in 4 volumes per `minio` pod.
+
+  If the cluster does not have sufficient Persistent Volume (`PV`) to bind to the
+  generated `PVC`, Tenant creation hangs until the required `PV` exist.
+
+- The `--capacity` field indicates the total capacity of the cluster. MinIO determines the
+  amount of storage to request for each `pvc` by dividing the specified capacity by the
+  total number of volumes in the server. The example above results in 1Ti requested
+  capacity per volume.
+
+  If the cluster does not have sufficient Persistent Volume (`PV`) to satsify the
+  requested capacity, Tenant creation hangs until the required `PV` exist.
```

Comment:
```suggestion
  requested capacity, Tenant creation hangs until the required `PV` exists.
```

Reply:
Same as above. I'm actually not sure of the exact grammar rules here (how do you pluralize an acronym anyways). Will chew on this.

Reply:
Tweaked as per above